.copywrite.hcl

```diff
@@ -0,0 +1,9 @@
+project {
+  license = "MPL-2.0"
+  copyright_year = 2021
+  header_ignore = [
+    "*.hcl2spec.go", # generated code specs, since they'll be wiped out until we support adding headers to generated files.
+    "**/test-fixtures/**",
+    "examples/**",
```

Comment:
So I see that the supervisor examples are actually nested within the builder directories so this requires a little change.

Reply:
`"**/examples/**"` ?

Reply:
done